tests/queries/test_q.py

```diff
@@ -225,6 +225,10 @@ def test_create_helper(self):
                     Q(*items, _connector=connector),
                 )
 
+    def test_fields(self):
+        q = Q(Q(field_1=1) & Q(field_2=1), field_2=1, field_3__lookup=True)
```

Comment:
It'd be great to have a more extensive coverage here that also deals with any non-`Q` arg such as `Lookup` instances provided to `Q`.